readme.markdown

```diff
@@ -407,6 +407,12 @@ With [npm](https://npmjs.org) do:
 npm install tape --save-dev
 ```
 
+# generating changelog
+
+```sh
+npx @r0mflip/logtag -r substack/tape -o CHANGELOG.md -t <github-token>
+```
```

Comment:
i definitely think this should be an `npm run-script`, and the dev dep added.

Reply:
I will move it to npm scripts and add as a devdep, but how do we manage the token in NPM scripts? Iam out of ideas how to do it.

Reply:
We'd just assume it's an available environment variable, and error out if it's not present.